include/backend/wayland.h

```diff
@@ -14,6 +14,15 @@
 #include <wlr/types/wlr_box.h>
 #include <wlr/render/drm_format_set.h>
 
+struct wlr_wl_seat {
+	struct wl_seat *wl_seat;
+
+	/* TODO: struct wl_list link; */
```

Comment:
is there a reason not to use `wl_list` here?

Reply:
This PR is only for factoring out stuff into structure that will be in the list in next https://github.com/swaywm/wlroots/pull/2417 that actually introduce multi-seat support.